File: core/src/main/java/com/exozet/android/core/interfaces/annotations/Transit.java
package com.exozet.android.core.interfaces.annotations;


import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import static androidx.fragment.app.FragmentTransaction.*;

@IntDef({TRANSIT_NONE, TRANSIT_FRAGMENT_OPEN, TRANSIT_FRAGMENT_CLOSE, TRANSIT_FRAGMENT_FADE})
@Retention(RetentionPolicy.SOURCE)
public @interface Transit {
}

File: core/src/main/java/com/exozet/android/core/interfaces/annotations/Transit.kt
package com.exozet.android.core.interfaces.annotations


import androidx.annotation.IntDef


import androidx.fragment.app.FragmentTransaction.*

@IntDef(TRANSIT_NONE, TRANSIT_FRAGMENT_OPEN, TRANSIT_FRAGMENT_CLOSE, TRANSIT_FRAGMENT_FADE)
@Retention(AnnotationRetention.SOURCE)
annotation class Transit